### PR TITLE
BF: Skip datatsets in fcptable pipeline

### PR DESCRIPTION
**Pull request by glalteva**
Temp workaround until redirect_url is complete- certain datasets have links which redirect to different url that must be crawled in order to grab tarballs rather than that original url being a tarball download itself.
Other datasets (Durham_Madden) have no links at all.
For now, will just skip those problematic datasets.

Working towards closing #655.

### CODE REVIEW

**Comment by glalteva** [comment]
@yarikoptic coveralls failed here too- but no untested code?

File: datalad/crawler/pipelines/fcptable.py [hunk 1]
```diff
@@ -20,12 +19,19 @@
...
+       URL point to all datasets, hence the URL at the top
+    """
     annex = Annexificator()
     lgr.info("Creating a FCP collection pipeline")
     return [
         crawl_url(url),
         xpath_match('//*[@class="tableHdr"]/td/strong/text()', output='dataset'),
-        # TODO: replace spaces
+        # skipping Cleveland and NewYork due to URL redirects, Durham due to lack of dataset tarball
+        skip_if({'dataset': 'Cleveland CCF | Durham_Madden | NewYork_Test-Retest_Reliability'}, re=True),
```
Comment (glalteva) [inline datalad/crawler/pipelines/fcptable.py]: delete spaces

Comment (yarikoptic) [inline datalad/crawler/pipelines/fcptable.py]: On Wed, 20 Jul 2016, Gergana Alteva wrote:
>    In datalad/crawler/pipelines/fcptable.py:
>  >      annex = Annexificator()
>  >      lgr.info("Creating a FCP collection pipeline")
>  >      return [
>  >          crawl_url(url),
>  >          xpath_match('//*[@class="tableHdr"]/td/strong/text()', output='dataset'),
>  > -        # TODO: replace spaces
>  > +        # skipping Cleveland and NewYork due to URL redirects, Durham due to lack of dataset tarball
>  > +        skip_if({'dataset': 'Cleveland CCF | Durham_Madden | NewYork_Test-Retest_Reliability'}, re=True),

>    delete spaces

and test (ok if just manually) if works as it should ;)

**Comment by yarikoptic** [comment]
great!

**Pull request merged**